### DISCUSSION TITLE
Strange behavior about connection between collections

### DISCUSSION THREAD

**Discussion by Dsantosjaime** [prisma/prisma#21457] [Q&A]
### Bug description

I mark this as a bug but It's help me at the moment.

I'm working with MongoDb and Prisma.

I have two collections, connected by a one to many relationship.
I use "createMany" and (as you know ?), It's not possible to connect multiple newly created documents to an other document.
Trying to glitch this limitation I just set all relationId of the new documents to the id of the document they should be in relation with.

And actually when I use "include" in other query, the documents are well connected.

What's the purpose of connect, if it's work like this ?

Can I continue to use this "glitch" safely ? If I can't I should use mongoose again because Prisma is to much limitating.

### How to reproduce

Connect two collection, not using  the key word "connect" but only setting the relation's ID, in the "many" part in a one to many relationship.

### Expected behavior

I want to know If I can use this technique or if it's just a coincidence.

### Prisma information

function : CreateMany

"@prisma/client": "^5.0.0",

### Environment & setup

NestJS / MongoDb / Prisma

### Prisma Version

```

```

**Comment by ludralph** [comment]
Hi @Dsantosjaime 👋 

Thank you for raising this question.

It is possible to set the relation `id` without using `connect`. This is documented [here](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#create-a-new-profile-record-and-connect-it-to-an-existing-user-record-via-an-id-field).

You always use `connect` keyword or set the foreign key field directly.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by Dsantosjaime** [reply]
Thanks for your answer, that's all I need to know, I didn't see it on the docs

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
